### ISSUE TITLE
Set S/I position of FOV relative to where the last scan was done. 

### ISSUE THREAD

**Issue by rpeltekov** [rpeltekov/ge3t_shim_tool#21]
need the shim tool to perform shimming at isocenter around where the user actually wants to scan. currently the shim tool prescribes a scan with a preset S/I position. it really needs to be set based on where the last scan was done. either extract this information from a previous scan, or just take the last set patient table position.

thanks to help from miki, found the information is logged to `/usr/g/service/log/irmJvm.log`, within lines like:
```
Sat 05/25/2024 10:53:03.473 AM PDT::::TablePositionApplication: Table Position=0
Sat 05/25/2024 10:53:03.521 AM PDT::::TablePositionApplication: Table Position=0
Sat 05/25/2024 10:53:03.581 AM PDT::::TablePositionApplication: Table Position=S1
Sat 05/25/2024 10:53:05.675 AM PDT::::TablePositionApplication: Table Position=S42
Sat 05/25/2024 10:53:05.681 AM PDT::::TablePositionApplication: Table Position=S42
```

Note: this then requires the user to do a scan in the position they desire to begin with first... or at the very leat they need to set their FOV, save scan, and then advance patient table so that information is then available to the shim GUI.

This is probably better done in junction with #8 and #16, or maybe directly followed up by them...

**Comment by rpeltekov** [comment]
consider that when doing this, the short-term goal is to be able to set the slice / center of the volume that you want to image at isocenter first. then you dont have to worry that the background will be obtained differently...

will need to design how to translate the linear gradient fields because as the scanner bed moves they will move relative to the desired slice.

up to now, i have had the scanner bed fixed for all slices that are being imaged...

**Comment by rpeltekov** [comment]
this fixes #14

**Comment by rpeltekov** [comment]
This will be completed following this **easy config goal**:
1. first the user will select their desired fov on the scanner. 
2. then they will move the scanner bed to that position
3. then the tool will simply use that scanner bed as the center of all of its future calibrations and shimming operations.
4. the current state of this quick solution means that you will not be able to change the position / thing you wish to scan without re-calibrating the shims. (that is until the linear gradients are paremetrized and their basis maps are not necessarily acquired every single time)
